src/ui/code/settingsPage.ts

```diff
@@ -208,7 +212,7 @@ export class settingsPage {
 		settingsPage.grdServers = jq("#grdServers").DataTable({
 			"dom": dataTableCustomDom,
 			paging: true,
-			select: true,
+			select: { style: "multi+shift" },
```

Comment:
With this select style , the edit updates are not reflected on the grid. Also the behavior is weird.
Select "os" works ok I think

Reply:
With "os" style, user might not know using "Ctrl" and "shift" to select multiple rows. :( 

(maybe a line of hint would better)